examples/typescript/platform-ref-aws/crossplane.ts

```diff
@@ -0,0 +1,77 @@
+import { App, Chart } from 'cdk8s';
+
+//import * as config from './crs/configuration.meta.pkg.crossplane.io';
+//import { Configuration } from 'crossplane-cdk';
+import { Configuration } from '../../../src';
+
+import * as xrds from './crs/xrd.apiextensions.crossplane.io';
+import * as comp from './crs/composition.apiextensions.crossplane.io';
+
+export function configYaml(crossplanePackage: App): Chart {
+
+    const yaml = new Chart(crossplanePackage, 'crossplane');
```

Comment:
Why is this called `yaml`?

Reply:
b/c each chart generates a yaml file and wanted naming that maps to other Crossplane examples to conceptually map what we're seeing esp. given the  demo flow.